### ISSUE TITLE
Wrapper AppDirs are indistinguishable from full AppDirs

### ISSUE THREAD

**Issue by moochris** [helloSystem/Filer#81]
The problem is that a user will think they are copying the current version of an application to another location to save it, or might want to copy it to a machine with no network connection.

Can we draw something on the icon automatically in Filer if the main executable is a link to a binary outside the AppDir?

**Comment by probonopd** [comment]
Yes, I think this is what "Emblems" are for. Like the little arrow for symlinks:

![image](https://user-images.githubusercontent.com/2480569/123691817-5a609c00-d856-11eb-8990-6a7f559a0067.png)

We could think about a similar symbol for wrapper AppDirs.
